src/test/java/net/sf/jabref/exporter/SaveActionsTest.java

```diff
@@ -36,14 +38,14 @@ public void checkSaveActionsParsing() throws IOException {
                 "  ISSN                     = {0886-5930}\n" +
                 "}\n" +
                 "\n" +
-                "@comment{jabref-meta: saveActions:title;IdentityFormatter;}"));
+                "@comment{jabref-meta: saveActions:enabled;title[IdentityFormatter]}"));
 
         ParserResult parserResult = parser.parse();
 
         List<String> saveActions = parserResult.getMetaData().getData(SaveActions.META_KEY);
 
-        assertEquals("title", saveActions.get(0));
-        assertEquals("IdentityFormatter", saveActions.get(1));
+        assertEquals("enabled", saveActions.get(0));
+        assertEquals("title[IdentityFormatter]", saveActions.get(1));
 
         SaveActions actions = new SaveActions(parserResult.getMetaData());
```

Comment:
Move to new test (and construct the save actions as well as the test entry by hand)